content/en/real_user_monitoring/guide/browser-sdk-upgrade.md

```diff
@@ -122,6 +122,12 @@ In v5, the default `sessionReplaySampleRate` is 0 instead of 100. If you don't i
 
 ### RUM
 
+### APM integration
+
+To promote the support and usage of OpenTelemetry, the default propagator types has been changed to include `tracecontext` in addition to `datadog`.
```

Comment:
should it be "default propagator types have*" or "default propagator type* has" instead? so the pluralization is in agreement